vendor.conf

```diff
@@ -23,6 +23,7 @@ golang.org/x/net master
 golang.org/x/text master
 github.com/docker/distribution master
 github.com/docker/libtrust master
+github.com/docker/docker-credential-helpers d68f9aeca33f5fd3f08eeae5e9d175edf4e731d1
```

Comment:
No wait you need your c/image fork as well

Reply:
Ah, I think I got it now.

Reply:
You dead yeah